docs/en/engines/database-engines/replicated.md

```diff
@@ -0,0 +1,50 @@
+
+# Replicated {#replicated}
+
+Supports replication of metadata via DDL log being written to ZooKeeper and executed on all of the replicas for a given database.
+One Clickhouse server can have multiple replicated databases running and updating at the same time.
+
+## Creating a Database {#creating-a-database}
+``` sql
+    CREATE DATABASE testdb ENGINE = Replicated('zoo_path', 'replica_name')
+```
+
+**Engine Parameters**
+
+-   `zoo_path` — ZooKeeper path. The same ZooKeeper path corresponds to the same database.
+-   `replica_name` — Replica name. Replica names must be different for all replicas of the same database.
+
+Using this engine, creation of Replicated tables requires no ZooKeeper path and replica name parameters. Table's replica name is the same as database replica name. Table's ZooKeeper path is a concatenation of database's ZooKeeper path, `/tables/`, and `UUID` of the table.
```

Comment:
As for now it's incorrect, behavior is the same as with `Atomic` engine. If no arguments provided for `ReplicatedMergeTree` engine, then default arguments are used: `/clickhouse/tables/{uuid}/{shard}` and `{replica}` (can be changed by server settings `default_replica_path` and `default_replica_name`). Macro `{uuid}` is unfolded to table's uuid, `{shard}` and `{replica}` are unfolded to values from server config, not from database engine arguments.

But I should change it and make `ReplicatedMergeTree` use shard and replica name of `Replicated` database.